examples/example.rb

```diff
@@ -15,6 +15,19 @@
 #                               request_headers: headers,
 #                               http_options: {open_timeout: 15, read_timeout: 30})
 
+# If you want to make request via proxy, you can set your proxy server in two ways.
+#
+# (1) Pass proxy_options hash
+#
+# client = SendGrid::Client.new(host: host,
+#                               request_headers: headers,
+#                               proxy_options: { host: '127.0.0.1', port: 8080 })
+#
+# (2) Set 'http_proxy' environment variable
```

Comment:
what about `https_proxy` ?

Reply:
No, you cannot use `https_proxy`. It is from ruby's [`URI::Generic#find_proxy'](http://ruby-doc.org/stdlib-2.5.1/libdoc/uri/rdoc/URI/Generic.html#method-i-find_proxy).